monkestation/code/modules/clothing/gloves/power_gloves.dm

```diff
@@ -0,0 +1,96 @@
+/obj/item/clothing/gloves/color/yellow/power_gloves
+	name = "Power Gloves"
+	desc = "Insulated gloves with onboard machinery that appears to be able to redirect the electrical current towards a creature."
+	armor_type = /datum/armor/power_gloves
+	var/datum/action/cooldown/spell/pointed/glove_zap/zap = new
+
+/obj/item/clothing/gloves/color/yellow/power_gloves/Destroy()
+    qdel(zap)
+    return ..()
+
+/datum/armor/power_gloves
+	acid = 50
+	bio = 50
+	fire = 80
+
+/datum/action/cooldown/spell/pointed/glove_zap
+	name = "Nerd Blast"
+	desc = "Blast your foes with the electricity surging beneath your feet!"
+	button_icon_state = "lightning"
+	cooldown_time = 10 SECONDS
+	spell_max_level = 1
+	sparks_amt = 4
+	spell_requirements = SPELL_REQUIRES_HUMAN
+	antimagic_flags = NONE
+	background_icon_state = ACTION_BUTTON_DEFAULT_BACKGROUND
+	overlay_icon_state = "bg_default_border"
+
+//had to recreate tesla zap into a pointed version
+/datum/action/cooldown/spell/pointed/glove_zap/proc/target_tesla_zap(atom/source, atom/target, power, zap_flags = ZAP_DEFAULT_FLAGS, max_damage = 90)
```

Comment:
i don't think `max_damage` is used?

Reply:
Oh you're right, forgot to move the infinity to max damage.

Reply:
I lied, max damage is already used in the glove's args.